components/switch/src/Switch.js

```diff
@@ -62,14 +48,143 @@ const asArray = val => (Array.isArray(val) ? val : [val]);
 // ==============================
 // Component
 // ==============================
+export const Switch = ({
+	name,
+	label,
+	checked: isChecked,
+	onChange,
+	size,
+	block,
+	flipped,
+	toggleText,
+	disabled,
+	srOnlyText,
```

Comment:
We should still spread the rest:

```suggestion
	srOnlyText,
	...rest,
```